app/assets/javascripts/modules/moj.AsyncGA.js

```diff
@@ -41,7 +60,20 @@
 
       // Configure profiles and make interface public
       // for custom dimensions, virtual pageviews and events
+      if (document.cookie.indexOf('accepted_cookies=true') > -1) {
+        this.hideCookieBanner();
+        window['ga-disable-UA-14565299-1'] = false;
+        this.trackPageView();
+      }
+      if (document.cookie.indexOf('accepted_cookies=false') > -1) {
+        this.hideCookieBanner();
+        window['ga-disable-UA-14565299-1'] = true;
+      }
 
+      this.trackPageView();
```

Comment:
Is the call to `this.trackPageView()` needed here?

If I'm reading this correctly, we already call `this.trackPageView()` on line 66 when the cookie `accepted_cookies=true` is set. So I'm wondering whether this second call, outside of that `if` statement, will result in duplicate page views being recorded?

Reply:
no it was not needed, Ive now removed this